### ISSUE TITLE
Screenreader does not read Notifications

### ISSUE THREAD

**Issue by felix-hcl** [felix-hcl/template_tests#2]
### Latest version

- [X] I have tested the latest version

### Steps to reproduce

Steps:
1. Trigger a Snackbar notification
2.Screen reader does not automatically read the content

### Current behavior

The screen reader does not read any snackbar notifications. This makes it harder to understand what is happening within an application.

### Expected behavior

The screen reader should treat those messages as an alert: `role="alert"` would do the trick

### Context

We want to be compliant with WCAG 2.1
